conformity/fields/country.py

```diff
@@ -27,17 +31,13 @@ def __init__(self, code_filter=lambda x: True, **kwargs):
 
     def errors(self, value):
         if not isinstance(value, six.text_type):
-            return [Error("Not a unicode string")]
+            return [Error('Not a unicode string')]
 
         if value not in self.values:
-            return [Error(
-                "Not a valid country code",
-                code=ERROR_CODE_INVALID,
-                pointer="value",
-            )]
+            return [Error('Not a valid country code')]
 
     def introspect(self):
-        return strip_none({
-            "type": self.introspect_type,
-            "valid_country_codes": self.values,
-        })
+        return {
+            'type': self.introspect_type,
+            'values': list(self.values),
+        }
```

Comment:
As above, do not change the quote style.

Reply:
Fixed.